README.md

```diff
@@ -93,7 +93,7 @@ $ cargo run --features json_example --example cli FILENAME.sql [--dialectname]
 ## Users
 
 This parser is currently being used by the [DataFusion] query engine,
-[LocustDB], [Ballista] and [GlueSQL].
+[LocustDB], [Ballista], [GlueSQL], and [Opteryx].
```

Comment:
@joocer thanks! 

@alamb I don't think there are any standards around projects that use others, so I'm not sure if that's ok.

Anyway, maybe this session could be renamed to 'Acknowledgements'? Users seems a bit weird IMO

Reply:
@AugustoFKL  I don't think "Acknowledgements" is quite right as we are not really acknowledging assistance from other projects. 

The point of this section, as I understand, is basically to encourage more use of the sqlparser crate by offering evidence that other projects have found it valuable enough to use. 

Perhaps we could change the heading (in a different PR) to something like 'Powered By' to follow https://arrow.apache.org/powered_by/ ?